packages/ai-hugging-face/src/node/huggingface-language-model.ts

```diff
@@ -27,12 +27,18 @@ import { CancellationToken } from '@theia/core';
 import { HfInference } from '@huggingface/inference';
 
 export const HuggingFaceModelIdentifier = Symbol('HuggingFaceModelIdentifier');
-
-function toHuggingFacePrompt(messages: LanguageModelRequestMessage[]): string {
+const createMessageContent = (message: LanguageModelMessage): string => {
+    if (LanguageModelMessage.isTextMessage(message)) {
+        return message.text;
+    }
+    return '';
+};
+function toHuggingFacePrompt(messages: LanguageModelMessage[]): string {
     if (messages.length === 1) {
-        return messages[0].query;
+        const message = messages[0];
+        return (LanguageModelMessage.isTextMessage(message) && message.text) || '';
     }
-    return messages.map(message => `${toRoleLabel(message.actor)}: ${message.query}`).join('\n');
+    return messages.map(message => `${toRoleLabel(message.actor)}: ${createMessageContent(message)}`).join('\n');
```

Comment:
Should empty messages be filtered?

Reply:
We are now filtering empty messages here, however we have the same pattern for all other language model implementations too. So if we adapt the behavior here, then we should adapt it for all others too.